### PR TITLE
Resolved some doctests, added TODO for setup.py and notes for people to add more doctests

### PR DESCRIPTION
**Pull request by WideEyedIngenue**

### CODE REVIEW

File: gender_novels/analysis/gender_pronoun_freq_analysis.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ def get_mean(data_dict):
     return mean_dict
 
 
-if __name__ == "__main__":
+#if __name__ == "__main__":
```
Comment (srisi) [inline gender_novels/analysis/gender_pronoun_freq_analysis.py]: Probably better, if no function specified, run tests.
```
if __name__ == '__main__':
    from dh_testers.testRunner import main_test
    main_test()
```

**Comment by mscuthbert** [comment]
Hi! would you mind putting your real name (or a hint of your real name) on your github account settings page, so we know with whom we're communicating. Thanks!

File: gender_novels/analysis/gender_pronoun_freq_analysis.py (hunk 1 above)
Comment (WideEyedIngenue) [reply gender_novels/analysis/gender_pronoun_freq_analysis.py]: I think I fixed it

**Comment by srisi** [comment]
Thank you!

**Pull request merged**